pkg/controller/servicebindingrequest/planner/planner.go

```diff
@@ -126,6 +126,7 @@ func (p *Planner) Plan() (*Plan, error) {
 
 // NewPlanner instantiate Planner type.
 func NewPlanner(
+	// FIXME: should we have only service-binding-request informing the namespace?
```

Comment:
umm.. what do you mean?

Reply:
I mean we should not have `namespace` (string) being informed to the types, we should instead use the SBR, and get it from the instance. So w make sure the SBR is the only authority to define the namespace.